packages/select/custom-resources/AvOrganizationSelect.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  avFilteredOrganizationsApi,
+  avOrganizationsApi,
+} from '@availity/api-axios';
+import { ResourceSelect } from '..';
+
+const AvOrganizationSelect = ({
+  name,
+  parameters: { permissionId = [], resourceIds = [], regionId } = {},
+  ...props
+}) => {
+  // Convert permissions and resources to array so that the dependency array is kept consistent
+  permissionId =
+    typeof permissionId === 'string' ? [permissionId] : permissionId;
+  resourceIds = typeof resourceIds === 'string' ? [resourceIds] : resourceIds;
+
+  const OrganizationSelect = React.useMemo(
+    () =>
+      ResourceSelect.create({
+        resource:
+          resourceIds.length > 0 && permissionId.length > 0
+            ? avFilteredOrganizationsApi
+            : avOrganizationsApi,
+        labelKey: 'name',
+      }),
+    // React will not be able to statically check if values are the same
```

Comment:
I think we should memoize the dependencies, too. Inspiration: "Dependencies Lists" section from this article: https://kentcdodds.com/blog/usememo-and-usecallback